apps/condo/domains/common/components/LayoutContext.tsx

```diff
@@ -34,6 +35,7 @@ export const LayoutContextProvider: React.FC = (props) => {
     }
 
     const isSmall = (breakpoints.md || breakpoints.xs || breakpoints.sm) && !breakpoints.lg
+    const tableScrollInLowResolution = (breakpoints.md || breakpoints.xs || breakpoints.sm) && !breakpoints.xl
```

Comment:
maybe `lowResScroll` ? It's anyway used within tables, so we can make it a bit shorter.

Reply:
Fixed

Reply:
I disagree. See my [comment below](#discussion_r855346923).